lib/google/object_store.rb

```diff
@@ -1,47 +0,0 @@
-# Copyright 2017 Google Inc.
```

Comment:
This is an unecessary file. It is not referenced anywhere in the codebase.

(grep for 'object_store')